searchobject/src/main/java/org/jalau/at18/searchobject/model/facedetector/FaceDetect.java

```diff
@@ -56,7 +56,7 @@ public static void faceDetection (String file, String type) throws FaceDetection
 
         //imgFile it's the file we want to analyze if there it's a person in the photo
         String imgFile = currentDir +"\\"+ file;
-
+        System.out.println("result " + imgFile);
```

Comment:
Please remove this `System.out.println()` if it's not necessary

Reply:
already removed